ipwb/indexer.py

```diff
@@ -52,7 +52,7 @@
 
 DEBUG = False
 
-IPFS_API = ipfsapi.Client(IPFSAPI_HOST, IPFSAPI_PORT)
+IPFS_API = ipfsapi.Client(f"/dns/{IPFSAPI_HOST}/tcp/{IPFSAPI_PORT}/http")
```

Comment:
I think it is time to retire `IPFSAPI_HOST` and `IPFSAPI_PORT` and replace them with a single config called `APFSAPI_ADDR` that will hold the multaddr value as described in the new API. The rationale behind this is the fact that the formation of address now depends on the format of the host. This change assumes that the `IPFSAPI_HOST` will be a hostname, but if it's an IP address, then this will render invalid. Consolidating complete ADDR in one string will make sure that the user modifies everything accordingly (and any misconfiguration is users' fault, not of the system).

Reply:
@ibnesayeed I agree that the semantics are off on HOST vs. IP. Let's update that in a separate optimization request with respect to #349, where it will be most useful. Please see #610. Until then, for the sake of integrating with the new version of the module, let's keep HOST and PORT for now, despite the lack of semantics.

Reply:
I would have made this change in this PR, but if you would prefer doing it in yet another PR, that is alright too.

Reply:
The changes in #610 will require additional parsing logic and assuring that they have been applied across more files. The ticket ought to be representative of the need to do it and will hopefully allow us to build some test cases around it, which seems like it would be scope creep for this PR/issue.